src/main/java/main/movie/dao/impl/OrderDaoImpl.java

```diff
@@ -0,0 +1,47 @@
+package main.movie.dao.impl;
+
+import java.util.List;
+import main.movie.dao.OrderDao;
+import main.movie.exceptions.DataProcessingException;
+import main.movie.lib.Dao;
+import main.movie.model.Order;
+import main.movie.model.User;
+import main.movie.util.HibernateUtil;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+
+@Dao
+public class OrderDaoImpl implements OrderDao {
+    @Override
+    public Order add(Order order) {
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.save(order);
+            transaction.commit();
+            return order;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Can't add order to DB! " + order, e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
+    }
+
+    @Override
+    public List<Order> getByUser(User user) {
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            return session.createQuery("FROM Order orders "
```

Comment:
if your order has more than one ticket won't you have duplications in your order list?